test/runtests.jl

```diff
@@ -30,12 +30,6 @@ simplerender(x) = replace(render(x), r"\s+" => ' ')
     end
 
     @testset "htl" begin
```

Comment:
Ensure we get 100% coverage, but make simpler tests. Examples should be in the documentation.